### PR TITLE
Update topic 4

### PR DESCRIPTION
**Pull request by andres-p28**

### CODE REVIEW

File: docs/topic4.md [hunk 1]
```diff
@@ -1,3 +1,170 @@
...
+In this Topic we will focus on learning how JavaScript approaches Object-Oriented programming and
+some patterns to group related functionality together.
+
+If you come from Java, or .NET you will find yourself a little bit lost at the beggining.
+ECMAScript6 provides a layer of syntactic sugar over the previous version (5.1) that is expected to simplify the language.
+But there aren't the classic classes from another Object-Oriented languages
+
+**Documentation**
+
+- [Object-oriented JS](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Object-oriented_JS)
```
Comment (maximilianotulian) [inline docs/topic4.md:12]: Bro, the documentation could be moved inside each topic? As it was with the other ones.

Let me know if you are busy to do it

File: docs/topic4.md [hunk 2]
```diff
@@ -1,3 +1,170 @@
...
+- Read the Chapter on JavaScript classes from [*Understanding ECMAScript6*](https://leanpub.com/understandinges6/read#leanpub-auto-classes)
+
+
+**Practice**
+
+1.  Creating classes
+
+    *   Create a Movie Class with the following structure
+
+        <table>
```
Comment (maximilianotulian) [inline docs/topic4.md:31]: I think the templates could be moved inside assets as the other one, makes sense?

File: docs/topic4.md [hunk 3]
```diff
@@ -1,3 +1,170 @@
...
+                    + constructor(name, age)
+                </td>
+            </tr>
+        </table>
+
+    *   Create a class called EventEmitter with the following structure
+
+        <table>
+            <tr>
+                <td>EventEmitter</td>
```
Comment (maximilianotulian) [inline docs/topic4.md:77]: Same with this ones

File: docs/topic4.md (hunk 1 above)
Comment (andres-p28) [reply docs/topic4.md:12]: Max, esto quedó así porque necesitas un poco de conocimiento de todo para arrancar este tópico.

File: docs/topic4.md (hunk 2 above)
Comment (andres-p28) [reply docs/topic4.md:31]: Esto no es código HTML, se esta usando para estilar el Markdown

**Pull request merged**